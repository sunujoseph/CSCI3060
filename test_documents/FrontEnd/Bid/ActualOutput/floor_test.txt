Enter Username: 
Error: You Do Not Have Privileges To Perform This Transaction
Error: Invalid Command
Error: Invalid Command
Error: Invalid Command
