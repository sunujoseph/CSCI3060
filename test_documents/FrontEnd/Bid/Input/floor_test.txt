login
userBS
bid
item2
userSS
10.01
