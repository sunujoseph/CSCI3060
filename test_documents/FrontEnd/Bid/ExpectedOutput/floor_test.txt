Enter Username: 
Enter Item Name: 
Enter Seller Username: 
Current Highest Bid Is: 12.34
Enter Bid: 
Error: Bid Must Be 5% Higher Than Current Highest Bid
Enter Bid: 